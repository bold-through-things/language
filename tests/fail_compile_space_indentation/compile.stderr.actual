[
  {
    "recoverable": false,
    "line": 4,
    "char": 0,
    "content": "    string \"world\"",
    "error": "oh spaces? yea? how many, two, or four, or maybe six?!",
    "error_type": "INVALID_INDENTATION"
  }
]
