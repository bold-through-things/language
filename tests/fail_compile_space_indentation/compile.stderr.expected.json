{
  "$items": [
    {
      "recoverable": false,
      "line": 4,
      "char": 0,
      "content": { "$contains": "string \"world\"" },
      "error_type": "INVALID_INDENTATION"
    }
  ],
  "$order": "any"
}